ccdaservice/utils/providers/providers.js

```diff
@@ -0,0 +1,59 @@
+const { fDate } = require('../date/date');
+const { cleanCode } = require('../clean-code/clean-code');
+const { NOT_INFORMED } = require('../constants');
+
+function populateProvider(provider, documentData) {
```

Comment:
Code looks good, but I am having some difficulty figuring out how the 2nd parameter in this function gets used in the current function calls in ccdaservice/serveccda.js .

Reply:
Ooops, I forgot to stage the changes for the function calls when committing. :see_no_evil:  Thanks for the catch!